src/main/frontend/components/block.cljs

```diff
@@ -1885,7 +1885,7 @@
        (not slide?)
        (merge attrs))
 
-     [:span
+     [:<>
```

Comment:
Replace the `span' wrapper with `Fragment'. This change also benefits the devtools selector, which would otherwise be difficult to select for this part (it is not a block element).